### PR TITLE
[TT-XLA][PyTorch] Training Workload for GPT-OSS-20B-Galaxy

### PR DESCRIPTION
**Pull request by abogdanovicTT**
**Issue**

 N/A

**Training Workload Setting**

- Model: GPT-OSS-20B
- Strategy: LoRA
- Framework: PyTorch
- Dataset: Alpaca
- Hardware: WH Galaxy

**What's Changed**

This PR introduces training of GPT-OSS-20B on WH Galaxy.

**Experiment Results**

<table>
  <tr>
     <td>Train Loss
     <td>Val Loss
  <tr>
  <tr>
    <td></td>
    <td></td>
  </tr>
</table>

**Checklist**

- [x]      Addition of appropriate .yaml file
- [x]      TT device testing
- [ ]      GPU testing

### CODE REVIEW

**Review by pglusacTT — APPROVED** [review]
🤟
